src/components/SampleItem.tsx

```diff
@@ -0,0 +1,393 @@
+    };
+    this.setState(state => ({ color: color }))
+    this.props.viewer.setBlendingInformation(blendingInformation);
+  }
+
+  handleColorGChange (
+    value: number,
+  ): void {
+    const identifier = this.props.opticalPathSequence.OpticalPathIdentifier
+    const blendInfo = this.props.viewer.getBlendingInformation
+      (identifier) as dmv.viewer.BlendingInformation
+    let color = [...blendInfo.color]
+    color[1] = value / 255.
+    const blendingInformation = {
+      color : color,
+      opticalPathIdentifier : identifier,
+    };
+    this.setState(state => ({ color: color }))
+    this.props.viewer.setBlendingInformation(blendingInformation);
+  }
+
+  handleColorBChange (
+    value: number,
+  ): void {
+    const identifier = this.props.opticalPathSequence.OpticalPathIdentifier
+    const blendInfo = this.props.viewer.getBlendingInformation
+      (identifier) as dmv.viewer.BlendingInformation
+    let color = [...blendInfo.color] 
+    color[2] = value / 255.
+    const blendingInformation = {
+      color : color,
+      opticalPathIdentifier : identifier,
+    };
+    this.setState(state => ({ color: color }))
+    this.props.viewer.setBlendingInformation(blendingInformation);
+  }
+
+  handleClippingChange (
+    value: number[],
+  ): void {
+    const identifier = this.props.opticalPathSequence.OpticalPathIdentifier
+    const blendingInformation = {
+      thresholdValues : value,
+      opticalPathIdentifier : identifier,
+    };
+    this.setState(state => ({ thresholdValues: value }))
+    this.props.viewer.setBlendingInformation(blendingInformation);
+  }
+
+  handleLimitChange (
+    value: number[],
+  ): void {
+    const identifier = this.props.opticalPathSequence.OpticalPathIdentifier
+    const blendingInformation = {
+      limitValues : value,
+      opticalPathIdentifier : identifier,
+    };
+    this.setState(state => ({ limitValues: value }))
+    this.props.viewer.setBlendingInformation(blendingInformation);
+  }
+
+  handleRemoveSample (): void {
+    const identifier = this.props.opticalPathSequence.OpticalPathIdentifier
+    this.props.viewer.deactivateOpticalPath(identifier)
+    this.props.itemRemoveHandler()
+  }
+
+  componentDidMount (): void {
+    const identifier = this.props.opticalPathSequence.OpticalPathIdentifier
+    const blendInfo = this.props.viewer.getBlendingInformation
+      (identifier) as dmv.viewer.BlendingInformation
+    this.setState(state => ({ visible: blendInfo.visible }))
+  }
+
+  render (): React.ReactNode {
+    const identifier = this.props.opticalPathSequence.OpticalPathIdentifier
+    const attributes: Array<{ name: string, value: string }> = []
+
+    const specimenDescription = this.props.specimenDescriptionSequence
+    if ('SpecimenShortDescription' in specimenDescription) {
+      const description = specimenDescription.SpecimenShortDescription
+      if (description) {
+        attributes.push({
+          name: 'Description',
```

Comment:
Better be explicit here, since it may otherwise not be clear what is being described:

```suggestion
          name: 'Specimen Description',
```